gulpfile.js

```diff
@@ -70,6 +72,11 @@ function handleError() {
   });
 }
 
+// cleanup ./dist in the initial build steps
+gulp.task('cleanup-dist', function (cb) {
+  del(['./dist/*'], cb);
+});
+
```

Comment:
Um, speaking of running things in parallel, I *think* that's what might happen if you add `cleanup-dist` to the `BUILD_TASKS` array... I might be wrong, but I think those tasks are all run in parallel, so it might cause a race condition! It might actually be easiest for now to just call the task `cleanup` and have it be run manually?